Makefile

```diff
@@ -166,9 +166,8 @@ VERSION_ASFLAGS := --defsym $(VERSION_DEF)=1
 # Stuff for showing the git hash in the intro on nightly builds
 # From https://stackoverflow.com/questions/44038428/include-git-commit-hash-and-or-branch-name-in-c-c-source
 ifeq ($(shell git rev-parse --abbrev-ref HEAD),nightly)
-  GIT_HASH=`git rev-parse --short HEAD`
-  COMPILE_TIME=`date -u +'%Y-%m-%d %H:%M:%S UTC'`
-  VERSION_CFLAGS += -DNIGHTLY -DGIT_HASH="\"$(GIT_HASH)\"" -DCOMPILE_TIME="\"$(COMPILE_TIME)\""
+  GIT_HASH := $(shell git rev-parse --short HEAD)
+  VERSION_CFLAGS += -DNIGHTLY -DGIT_HASH="\"$(GIT_HASH)\""
```

Comment:
COMPILE_TIME not used anywhere is source (looks to have come from stackoverflow copy/paste)